.github/workflows/aws_tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           aws-region: us-east-1
       - name: Start EC2 runner
         id: start-ec2-runner
-        uses: machulav/ec2-github-runner@v2.2.0
+        uses: machulav/ec2-github-runner@v2.3.0
```

Comment:
`machulav/ec2-github-runner` is used twice in this file so I think it makes to update both locations

Reply:
Missed that lol ! Thanks !